lib/zss.x

```diff
@@ -0,0 +1,256 @@
+ IMPORT CODE,'zssServer','readByte'                                            
+ IMPORT CODE,'zssServer','readCommentTail'                                     
+ IMPORT CODE,'zssServer','recoveryDisableCurrentState'                         
+ IMPORT CODE,'zssServer','recoveryEnableCurrentState'                          
+ IMPORT CODE,'zssServer','recoveryEstablishRouter'                             
+ IMPORT CODE,'zssServer','recoveryGetABENDCode'                                
+ IMPORT CODE,'zssServer','recoveryIsRouterEstablished'                         
+ IMPORT CODE,'zssServer','recoveryPop'                                         
+ IMPORT CODE,'zssServer','recoveryPush'                                        
+ IMPORT CODE,'zssServer','recoveryRemoveRouter'                                
+ IMPORT CODE,'zssServer','recoverySetDumpTitle'                                
+ IMPORT CODE,'zssServer','recoverySetFlagValue'                                
+ IMPORT CODE,'zssServer','recoveryUpdateRouterServiceInfo'                     
+ IMPORT CODE,'zssServer','recoveryUpdateStateServiceInfo'                      
+ IMPORT CODE,'zssServer','removeLocalLoggingContext'                           
+ IMPORT CODE,'zssServer','removeLoggingContext'                                
+ IMPORT CODE,'zssServer','reportJSONDataProblem'                               
+ IMPORT CODE,'zssServer','requestIntHeader'                                    
+ IMPORT CODE,'zssServer','requestStringHeader'                                 
+ IMPORT CODE,'zssServer','respondWithChunkedOutputStream'                      
+ IMPORT CODE,'zssServer','respondWithError'                                    
+ IMPORT CODE,'zssServer','respondWithJsonError'                                
+ IMPORT CODE,'zssServer','respondWithJsonPrinter'                              
+ IMPORT CODE,'zssServer','respondWithUnixDirectory'                            
+ IMPORT CODE,'zssServer','respondWithUnixFileContents'                         
+ IMPORT CODE,'zssServer','respondWithUnixFileContents2'                        
+ IMPORT CODE,'zssServer','respondWithUnixFileContentsWithAutocvtMode'          
+ IMPORT CODE,'zssServer','respondWithUnixFileNotFound'                         
+ IMPORT CODE,'zssServer','respondWithXmlPrinter'                               
+ IMPORT CODE,'zssServer','runServiceThread'                                    
+ IMPORT CODE,'zssServer','safeRealloc'                                         
+ IMPORT CODE,'zssServer','serveFile'                                           
+ IMPORT CODE,'zssServer','serveSimpleTemplate'                                 
+ IMPORT CODE,'zssServer','setConfiguredProperty'                               
+ IMPORT CODE,'zssServer','setContentType'                                      
+ IMPORT CODE,'zssServer','setDefaultJSONRESTHeaders'                           
+ IMPORT CODE,'zssServer','setHttpAuthTrace'                                    
```

Comment:
I'd blacklist all the tracers... doesn't seem like something that plugins should be touching.